src/app/productManagement/pricing/templates/createProductPrice.modal.html

```diff
@@ -1,4 +1,4 @@
-<form name="createProductPriceModal.form" autocomplete="off" novalidate pretty-submit>
+<form name="createProductPriceModal.form" ng-submit="createProductPriceModal.submit()" autocomplete="off" novalidate oc-pretty-submit>
```

Comment:
Did you test out that making these changes didn't mess up the price-breaks directives?